src/BayesFilters/include/BayesFilters/WhiteNoiseAcceleration.h

```diff
@@ -36,7 +36,7 @@ class bfl::WhiteNoiseAcceleration : public StateModel
 
     Eigen::MatrixXf getNoiseSample(const int num);
 
-    Eigen::MatrixXf getNoiseCovarianceMatrix() override;
+    Eigen::MatrixXf getNoiseCovarianceMatrix();
```

Comment:
If it does not override, can we move it to `protected`?

Reply:
At the end of this PR  `WhiteNoiseAcceleration` inherits from `LinearStateModel` that in turn inherits from `AdditiveStateModel`. Then the method is changed again to `override` in a next commit.